### ISSUE TITLE
3.6, 3.7, 3.8, 3.9 on PyPI

### ISSUE THREAD

**Issue by josephernest** [jamesls/semidbm#22]
Hello,
Congrats for this nice project.

    pip install semidbm

does not work on Python 3.7. 

Is it planned to update it on PyPI?

Thanks!

**Comment by besfahbod** [comment]
Fyi, we had to fork this project to add support for latest PY 3 editions. You can find the git repo at https://github.com/quora/semidbm2 . I believe we have not published it to pypi yet. Would adding git repo as dependency work for you?

**Comment by josephernest** [comment]
Yes @besfahbod, it would be a nice option, easy to install in one line! What is the installation command? (and maybe include it in the README?). I imagine it's something like:

    pip install git+https://github.com/.../.../...

**Comment by besfahbod** [comment]
Cool! Yeah, something like that should work. (https://pip.pypa.io/en/stable/reference/pip_install/#git)

We were thinking about publishing the fork under the name `semidbm2` on pypi. If you think that's useful for you, would appreciate any help with changes needed in the code to be able to do that. (Of course, any deps using the package need to be updated, too.)
